### ISSUE TITLE
🐛💄 `frontmatter` not properly handled by `typedoc`

### ISSUE THREAD

**Issue by lukas-runge** [oshifty/vision#23]
Basically this just concerns `README.md`. All other files are not integrated into `typedoc` anyways. This file needs to have `frontmatter` for `mdsvex` for the endeavour in #15 to work out. 🔥😇

`Typedoc` should just ignore `frontmatter` - currently it is shown like so:

<img width="259" alt="Bildschirmfoto 2023-08-29 um 16 47 17" src="https://github.com/oshifty/vision/assets/21328241/50d03bd1-1882-4535-8b1b-6e7efc56813b">

**Comment by lukas-runge** [comment]
As we use the `typedoc --watch` command for developing locally I would prefer fixing it in there. I haven't looked at it in more detail tho, so I can't evaluate if thats a practical solution. If we find it impractical then the quick fix is fine too as it will get the job done for now. 👌

**Comment by hrueger** [comment]
OK, I did some research. According to their [`package.json`](https://github.com/TypeStrong/typedoc/blob/master/package.json) it seems like `typedoc` is using [`marked`](https://github.com/markedjs/marked) as the markdown parser.
They have a (closed) issue about yaml frontmatter here: https://github.com/markedjs/marked/issues/485
TLDR: won't be implemented, out of scope.

Should we create a new issue in the [typedoc](https://github.com/TypeStrong/typedoc) repo?

**Comment by lukas-runge** [comment]
Yeah, let's create a new issue over in `typedoc`. This could be as easy as removing the frontmatter before parsing. This could be a feature enabled thru the `typedoc.json`. Let's see what they think. 👍

**Comment by hrueger** [comment]
Sure, see https://github.com/TypeStrong/typedoc/issues/2381

**Comment by hrueger** [comment]
fixed by https://github.com/TypeStrong/typedoc/issues/2381. Theme was also updated to support the new typedoc version in https://github.com/mengxinssfd/typedoc-theme/issues/3
